clients/ui/frontend/src/shared/style/MUI-theme.scss

```diff
@@ -55,6 +55,12 @@
   --mui-radio__input--Width: 10px;
   --mui-radio__input--Height: 10px;
 
+  // Sidebar from https://github.com/kubeflow/kubeflow/blob/4275d99754ac91f6cf5654b03824a73825e9fe55/components/centraldashboard/public/components/main-page.css#L7C1-L13C51 
+  --kf-central-primary-background-color: #0a3b71;
+  --kf-central-sidebar-default-color: #ffffff90;
```

Comment:
Had to update this color from the default value used in Kubeflow to fix the a11y error - is https://github.com/kubeflow/kubeflow/issues a good place to log this in order to contribute it back? @lucferbux @Griffin-Sullivan ?

Reply:
Yes, that might be a good place to talk about that. Approving this though, we can continue the conversation there.